File: resilience/src/main/java/yg0r2/examples/resilience/service/web/DummyRestController.java
package yg0r2.examples.resilience.service.web;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.UUID;

@RestController
@RequestMapping(value = "/api/dummy")
public class DummyRestController {

    @GetMapping
    public String greeting() {
        return UUID.randomUUID().toString();
    }

}

File: spring/feign/application/src/main/kotlin/yg0r2/example/spring/feign/application/web/DummyRestController.kt
package yg0r2.example.spring.feign.application.web

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RequestMapping("/api/dummy")
@RestController
class DummyRestController {

    @GetMapping
    fun getValues(): List<String> =
        listOf(
            "value - 1",
            "value - 2"
        )

}
